core/errors.go

```diff
@@ -29,6 +30,8 @@ func isTemporary(info httperror.Info, err error) bool {
 		return true
 	case "CH001": // request timed out
 		return true
+	case "CH012": // conflict
+		return true
```

Comment:
Makes sense.

Yeah eventually we'll prob need to distinguish between "your operation conflicted" and "an internal operation conflicted and you should retry".

Currently we're only sending the latter. In fact, should this case be a 5xx error?